pulsar/producer_impl.go

```diff
@@ -61,7 +60,7 @@ type producer struct {
 	ticker        *time.Ticker
 	tickerStop    chan struct{}
 
-	log *log.Entry
```

Comment:
Same excessive field rename. Can you please reduce review scope? ^^

Reply:
as mentioned above, i think using `logger` as field name is more idiomatic.